spec/realtime/message.test.js

```diff
@@ -70,6 +70,50 @@ define(['ably', 'shared_helper'], function(Ably, helper) {
 		}
 	};
 
+	/*
+	 * Test that a message is not sent back to the same realtime client
+	 * when echoMessages is false (RTC1a and RTL7f)
+	 *
+	 * Test that a message is sent back to the same realtime client
+	 * when echoMessages is true (RTC1a and RTL7f)
+	 */
+	exports.publishEcho = function(test) {
+		test.expect(3);
+
+		// set up two realtimes
+		var rt1 = helper.AblyRealtime({ echoMessages: false }),
```

Comment:
Instead of rt1 and rt2, why not name them intuitively such as `realtimeNoEcho` and `realtimeEcho`?